### PR TITLE
New cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-instrument-cal-csvs-dt-69)

File: calibration/OPTAAD/CGINS-OPTAAD-00245__20200316.csv [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+ACS-245,CC_tbins,"[1.682809, 2.286420, 3.499630, 4.439231, 5.562222, 6.506667, 7.494091, 8.486000, 9.498000, 10.486250, 11.490417, 12.501200, 13.486000, 14.486400, 15.492083, 16.478750, 17.499600, 18.485000, 19.505200, 20.507917, 21.493200, 22.482692, 23.502400, 24.479583, 25.480000, 26.477692, 27.512692, 28.474400, 29.512400, 30.502222, 31.482800, 32.494074, 33.502692, 34.469259, 35.000000]",
+ACS-245,CC_taarray,SheetRef:CC_taarray,
+ACS-245,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (dbogorff) [inline calibration/OPTAAD/CGINS-OPTAAD-00245__20200316.csv:9]: Looks good!

File: calibration/OPTAAD/CGINS-OPTAAD-00245__20200316__CC_taarray.ext [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+-0.008193,-0.00607,-0.005819,-0.005537,-0.005238,-0.005155,-0.004886,-0.00462,-0.004466,-0.004245,-0.003926,-0.003752,-0.003483,-0.003359,-0.003075,-0.002831,-0.002533,-0.002174,-0.001942,-0.00157,-0.001187,-0.000958,-0.00062,-0.000188,0,0.000202,0.000491,0.00078,0.000937,0.00122,0.001109,0.001255,0.001198,0.001276,0.001035
+-0.008264,-0.00619,-0.005887,-0.005665,-0.005319,-0.005264,-0.004892,-0.004659,-0.004401,-0.004233,-0.003931,-0.003733,-0.003529,-0.003446,-0.003106,-0.002796,-0.002583,-0.002271,-0.001977,-0.001617,-0.00115,-0.000958,-0.000628,-0.00016,0,0.00025,0.000548,0.000834,0.000936,0.001289,0.001202,0.001334,0.00123,0.001318,0.00077
+-0.009167,-0.006883,-0.006464,-0.006148,-0.005872,-0.005671,-0.005408,-0.005082,-0.004825,-0.004579,-0.004177,-0.003991,-0.003807,-0.003673,-0.003273,-0.002901,-0.002649,-0.002342,-0.002113,-0.001712,-0.001256,-0.001087,-0.000609,-0.000128,0,0.000276,0.000602,0.000853,0.00101,0.001445,0.001296,0.001576,0.001396,0.001507,0.000951
```
Comment (dbogorff) [inline calibration/OPTAAD/CGINS-OPTAAD-00245__20200316__CC_taarray.ext:89]: Used Notepad++ Compare Plugin.  Looks good!

File: calibration/OPTAAD/CGINS-OPTAAD-00245__20200316__CC_tcarray.ext [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+-0.001448,-0.001078,-0.00082,-0.000541,-0.000524,-0.000373,-0.000365,-0.000218,-0.000494,-0.000549,-0.000588,-0.000861,-0.000733,-0.00089,-0.000971,-0.000801,-0.00069,-0.000628,-0.000447,-0.000305,-0.000295,-0.000176,-2.4e-05,-1.7e-05,0,2.7e-05,9.9e-05,-0.000123,-3.3e-05,7.6e-05,3.6e-05,8.9e-05,7.1e-05,0.000127,8.7e-05
+-0.001141,-0.000849,-0.000691,-0.00066,-0.000579,-0.000296,-0.000347,-0.000337,-0.000463,-0.000553,-0.000575,-0.000709,-0.000709,-0.000706,-0.000644,-0.00043,-0.000457,-0.000305,-0.000153,-0.000156,-0.000115,1.7e-05,-8.5e-05,-4.9e-05,0,-7.5e-05,2.4e-05,0.000113,6.8e-05,2e-06,-1.9e-05,4.2e-05,6.3e-05,0.000131,-0.000165
+-0.001033,-0.001079,-0.000785,-0.000549,-0.000454,-8.3e-05,-3.6e-05,1.5e-05,-0.000194,-0.00026,-0.000319,-0.000548,-0.000596,-0.000604,-0.000665,-0.000508,-0.000586,-0.000499,-0.00044,-0.000351,-0.000218,-0.000218,-0.000102,-2.7e-05,0,-0.000145,-5.3e-05,-0.00017,-5.5e-05,6.2e-05,6.2e-05,3.1e-05,0.000145,7.8e-05,0.000742
```
Comment (dbogorff) [inline calibration/OPTAAD/CGINS-OPTAAD-00245__20200316__CC_tcarray.ext:89]: Used Notepad++ Compare Plugin.  Looks good!

File: calibration/PCO2WB/CGINS-PCO2WB-C0098__20200420.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0098,CC_cala,0.1797,source_file PCO2W-B_SAMI-CO2_SN_C0098_Calibration_Certificate_2020-04-20.pdf
+C0098,CC_calb,-0.1152,
+C0098,CC_calc,-0.5192,
+C0098,CC_calt,14.6766,
+C0098,CC_ea434,19706,constant
+C0098,CC_ea620,34,constant
+C0098,CC_eb434,3073,constant
+C0098,CC_eb620,44327,constant
```
Comment (dbogorff) [inline calibration/PCO2WB/CGINS-PCO2WB-C0098__20200420.csv:9]: Looks good!

File: calibration/PCO2WC/CGINS-PCO2WC-C0103__20200420.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0103,CC_cala,0.1894,source_file PCO2W-C_SAMI-CO2_SN_C0103_Calibration_Certificate_2020-04-20.pdf
+C0103,CC_calb,-0.1777,
+C0103,CC_calc,-0.4176,
+C0103,CC_calt,14.7016,
+C0103,CC_ea434,19706,constant
+C0103,CC_ea620,34,constant
+C0103,CC_eb434,3073,constant
+C0103,CC_eb620,44327,constant
```
Comment (dbogorff) [inline calibration/PCO2WC/CGINS-PCO2WC-C0103__20200420.csv:9]: Looks good!

File: calibration/PCO2WC/CGINS-PCO2WC-C0106__20200322.csv [hunk 6]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0106,CC_cala,0.0867,source_file PCO2W-C_SAMI-CO2_SN_C0106_Calibration_Certificate_2020-03-22.pdf
+C0106,CC_calb,0.4756,
+C0106,CC_calc,-1.4054,
+C0106,CC_calt,18.1005,
+C0106,CC_ea434,19706,constant
+C0106,CC_ea620,34,constant
+C0106,CC_eb434,3073,constant
+C0106,CC_eb620,44327,constant
```
Comment (dbogorff) [inline calibration/PCO2WC/CGINS-PCO2WC-C0106__20200322.csv:9]: Looks good!

File: calibration/OPTAAD/CGINS-OPTAAD-00240__20200326.csv [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+ACS-240,CC_tbins,"[0.743714, 1.385667, 2.512250, 3.630667, 4.493750, 5.501304, 6.498000, 7.509583, 8.471739, 9.518000, 10.491429, 11.491111, 12.505000, 13.499200, 14.500000, 15.481200, 16.487600, 17.485417, 18.491739, 19.473333, 20.505600, 21.443077, 22.520455, 23.549130, 24.484583, 25.426400, 26.493333, 27.410800, 28.539565, 29.540833, 30.468667, 31.507727, 32.538800, 33.530000, 34.400000]",
+ACS-240,CC_taarray,SheetRef:CC_taarray,
+ACS-240,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (batryn) [inline calibration/OPTAAD/CGINS-OPTAAD-00240__20200326.csv:9]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00241__20200326.csv [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+ACS-241,CC_tbins,"[0.949194, 1.219447, 2.505200, 3.441429, 4.512667, 5.486957, 6.504783, 7.524583, 8.512609, 9.511304, 10.503750, 11.493750, 12.482500, 13.482400, 14.494583, 15.492800, 16.475417, 17.502400, 18.490000, 19.467200, 20.511154, 21.512917, 22.486667, 23.482800, 24.494800, 25.476250, 26.483600, 27.498846, 28.488800, 29.490000, 30.500370, 31.519200, 32.478077, 33.497407, 34.478077]",
+ACS-241,CC_taarray,SheetRef:CC_taarray,
+ACS-241,CC_tcarray,SheetRef:CC_tcarray,
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/OPTAAD/CGINS-OPTAAD-00241__20200326.csv:9]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00245__20200316.csv (hunk 1 above)
Comment (batryn) [reply calibration/OPTAAD/CGINS-OPTAAD-00245__20200316.csv:9]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00245__20200316__CC_tcarray.ext (hunk 3 above)
Comment (batryn) [reply calibration/OPTAAD/CGINS-OPTAAD-00245__20200316__CC_tcarray.ext:89]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00245__20200316__CC_taarray.ext (hunk 2 above)
Comment (batryn) [reply calibration/OPTAAD/CGINS-OPTAAD-00245__20200316__CC_taarray.ext:89]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00241__20200326__CC_taarray.ext [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+-0.017877,-0.015174,-0.01214,-0.011455,-0.010883,-0.010298,-0.009461,-0.008781,-0.008182,-0.007608,-0.006896,-0.006222,-0.005701,-0.005027,-0.004576,-0.004292,-0.003393,-0.003095,-0.002716,-0.002286,-0.001761,-0.001519,-0.001041,-0.000676,-0.000395,0,4.7e-05,8e-05,0.000251,0.000346,0.000293,0.000363,0.000457,8.3e-05,0.000294
+-0.017757,-0.015046,-0.012052,-0.011312,-0.010526,-0.01014,-0.009339,-0.00867,-0.008093,-0.00744,-0.006933,-0.006146,-0.005482,-0.004924,-0.004488,-0.004128,-0.003237,-0.003006,-0.002595,-0.002203,-0.001737,-0.00141,-0.000959,-0.000631,-0.000325,0,6.2e-05,0.000127,0.000362,0.000479,0.00041,0.00056,0.000512,7.9e-05,0.000381
+-0.017722,-0.014991,-0.012023,-0.011322,-0.010569,-0.010149,-0.009295,-0.008738,-0.008064,-0.007461,-0.006828,-0.006032,-0.005547,-0.004862,-0.004477,-0.004101,-0.003236,-0.003003,-0.002626,-0.002262,-0.001678,-0.00143,-0.000909,-0.000605,-0.00033,0,0.000159,0.000147,0.000352,0.000485,0.000392,0.000525,0.000517,0.000222,0.000417
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/OPTAAD/CGINS-OPTAAD-00241__20200326__CC_taarray.ext:87]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00241__20200326__CC_tcarray.ext [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+0.002568,0.004039,0.004575,0.004422,0.004371,0.004294,0.004389,0.00441,0.004242,0.004149,0.003702,0.003342,0.0028,0.002383,0.00214,0.001847,0.001517,0.001089,0.000553,0.000484,-0.000183,-0.000167,-0.000109,-7.9e-05,3.8e-05,0,2.4e-05,0.000129,-4e-06,0.000124,0.000388,0.000472,0.000702,0.00083,0.000866
+0.001898,0.003808,0.005056,0.005188,0.005431,0.005354,0.005186,0.00496,0.005034,0.004863,0.004421,0.003751,0.003305,0.002883,0.002653,0.002244,0.001954,0.001506,0.000874,0.000686,-3e-06,1.5e-05,-3e-06,6.8e-05,6.6e-05,0,0.000204,5.4e-05,0.000141,4.9e-05,0.000375,0.000488,0.000742,0.000773,0.000867
+0.002553,0.004568,0.005429,0.0053,0.005236,0.004942,0.004915,0.004894,0.004595,0.004327,0.003909,0.00338,0.002885,0.002573,0.002226,0.001833,0.00152,0.001056,0.000481,0.000364,-0.000317,-0.000246,-0.000123,8.2e-05,8.8e-05,0,-8e-05,9.1e-05,1.8e-05,0.000142,0.00046,0.000492,0.000778,0.000804,0.000906
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/OPTAAD/CGINS-OPTAAD-00241__20200326__CC_tcarray.ext:87]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00240__20200326__CC_taarray.ext [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+0.000759,0.001414,0.001511,0.001567,0.001548,0.001451,0.001392,0.001378,0.001262,0.00115,0.001046,0.000915,0.000692,0.000544,0.000459,0.00019,7.6e-05,-5.9e-05,-0.000113,-0.000172,-0.000172,-0.000209,-0.000185,-0.000138,-5.6e-05,0,0.0002,0.000164,0.000162,0.00034,0.000402,0.000571,0.000638,0.000635,0.000659
+0.000574,0.001276,0.001348,0.001393,0.001414,0.001265,0.001206,0.001157,0.001085,0.00094,0.000873,0.000761,0.00063,0.000357,0.000379,2.9e-05,-2.6e-05,-0.000174,-0.000215,-0.000191,-0.000244,-0.000316,-0.00023,-0.000222,-4.5e-05,0,0.000206,0.000217,0.000165,0.000392,0.00041,0.000667,0.000661,0.000723,0.000806
+0.000381,0.001078,0.001163,0.001155,0.001242,0.001095,0.001069,0.001078,0.000951,0.000841,0.00075,0.000628,0.000411,0.00025,0.000196,-0.000106,-8.4e-05,-0.000304,-0.000336,-0.000286,-0.000314,-0.000348,-0.000222,-0.000226,-8.1e-05,0,0.000296,0.00021,0.000158,0.000457,0.00052,0.000706,0.000728,0.000827,0.000844
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/OPTAAD/CGINS-OPTAAD-00240__20200326__CC_taarray.ext:79]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00240__20200326__CC_tcarray.ext [hunk 12]
```diff
@@ -0,0 +1,79 @@
...
+-0.002735,-0.002496,-0.002442,-0.002308,-0.002152,-0.00184,-0.001886,-0.001716,-0.001665,-0.0016,-0.00149,-0.001382,-0.001247,-0.000958,-0.000722,-0.000611,-0.000198,-0.000265,-4.6e-05,-2e-05,0.000114,0.000201,5.6e-05,9.9e-05,4e-06,0,-0.000146,-0.000221,-0.000325,-0.000568,-0.000655,-0.000866,-0.000881,-0.00082,-0.00097
+-0.003183,-0.001543,-0.001312,-0.001163,-0.001236,-0.001105,-0.00102,-0.000734,-0.000568,-0.00031,-0.000339,-0.00012,-0.000196,-0.000212,-6.7e-05,-0.000231,-5e-05,-5.4e-05,7.3e-05,5.6e-05,0.000119,0.000116,0.000149,-0.000118,-7.9e-05,0,-7.2e-05,-0.000134,-0.000132,-0.000359,-0.000395,-0.000601,-0.000699,-0.000776,-0.001052
+-0.004646,-0.001664,-0.001513,-0.001542,-0.001338,-0.001658,-0.001663,-0.001678,-0.001606,-0.001377,-0.001004,-0.000673,-0.000228,5.6e-05,0.000248,0.000216,0.000632,0.00076,0.000647,0.000676,0.000714,0.000727,0.000401,0.000445,0.000209,0,-0.000154,-0.000316,-0.000231,-0.00039,-0.000598,-0.000711,-0.000822,-0.001312,-0.00108
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/OPTAAD/CGINS-OPTAAD-00240__20200326__CC_tcarray.ext:79]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00218__20200304__CC_taarray.ext [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+-0.002058,-0.004375,-0.00475,-0.004755,-0.004943,-0.005263,-0.005039,-0.004774,-0.004641,-0.004423,-0.003957,-0.003554,-0.003203,-0.002827,-0.002711,-0.002638,-0.002076,-0.001795,-0.001537,-0.001222,-0.001086,-0.000941,-0.000499,-0.000449,0,0.000178,0.00045,0.000685,0.001035,0.001198,0.001432,0.001662,0.001946,0.001913,0.001899
+-0.002222,-0.004615,-0.00503,-0.005022,-0.005144,-0.005504,-0.005281,-0.004871,-0.004845,-0.004583,-0.004018,-0.003634,-0.003269,-0.002881,-0.002694,-0.002613,-0.002013,-0.001933,-0.001526,-0.001298,-0.001121,-0.00099,-0.00052,-0.000352,0,0.000315,0.000417,0.000611,0.001109,0.001337,0.001507,0.001685,0.001994,0.00195,0.002357
+-0.002296,-0.004657,-0.004981,-0.005097,-0.005211,-0.005504,-0.005302,-0.004872,-0.004864,-0.004527,-0.004132,-0.003713,-0.003269,-0.002967,-0.002728,-0.002642,-0.002049,-0.001972,-0.001619,-0.001329,-0.001191,-0.000953,-0.000623,-0.000327,0,0.000179,0.00042,0.000646,0.001076,0.001311,0.001613,0.001727,0.002034,0.001978,0.001872
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/OPTAAD/CGINS-OPTAAD-00218__20200304__CC_taarray.ext:87]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00218__20200304__CC_tcarray.ext [hunk 14]
```diff
@@ -0,0 +1,87 @@
...
+0.000616,0.002801,0.003226,0.003178,0.003157,0.002992,0.002605,0.002308,0.002182,0.001845,0.001823,0.001678,0.001412,0.001423,0.00128,0.00112,0.000854,0.000721,0.000683,0.000557,0.000442,0.000295,0.00015,0.000245,0,-0.0001,-0.000135,-0.000115,-0.000235,-0.00031,-0.000359,-0.000542,-0.000805,-0.001013,-0.000897
+0.000554,0.002356,0.002584,0.002664,0.002589,0.002199,0.001846,0.001736,0.001535,0.001422,0.001315,0.001094,0.000814,0.00098,0.000726,0.000694,0.00042,0.00038,0.000334,0.00039,0.000258,0.000268,0.000209,0.000309,0,-0.000164,-0.000226,-0.000302,-0.000326,-0.000379,-0.000477,-0.000663,-0.000902,-0.000915,-0.001209
+-0.001042,0.001504,0.001734,0.001699,0.001707,0.001257,0.001022,0.001019,0.000984,0.000917,0.00095,0.000963,0.000712,0.000821,0.0007,0.00053,0.000381,0.000294,0.000272,0.000201,0.000186,0.000196,0.000155,0.000198,0,-4.3e-05,-0.000215,-0.000232,-0.000294,-0.000433,-0.000406,-0.00061,-0.000867,-0.001026,-0.000941
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/OPTAAD/CGINS-OPTAAD-00218__20200304__CC_tcarray.ext:87]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00193__20200302__CC_taarray.ext [hunk 15]
```diff
@@ -0,0 +1,85 @@
...
+-0.010475,-0.008773,-0.007798,-0.007309,-0.007069,-0.006734,-0.006194,-0.005979,-0.005369,-0.005012,-0.004948,-0.004754,-0.004341,-0.003842,-0.003634,-0.003145,-0.002861,-0.002622,-0.002392,-0.002101,-0.001648,-0.001299,-0.000945,-0.000397,0,0.000414,0.000675,0.001199,0.001536,0.00202,0.002324,0.002563,0.002634,0.002716,0.002724
+-0.010603,-0.008923,-0.007884,-0.007459,-0.007128,-0.006876,-0.006371,-0.00614,-0.005478,-0.005123,-0.004962,-0.004835,-0.004411,-0.004006,-0.003675,-0.003239,-0.002907,-0.002726,-0.002421,-0.002061,-0.00179,-0.001319,-0.000975,-0.000498,0,0.000298,0.000557,0.00114,0.001437,0.001834,0.002091,0.002237,0.00247,0.002511,0.002266
+-0.010598,-0.00887,-0.007879,-0.007508,-0.007107,-0.006877,-0.006273,-0.006077,-0.005434,-0.004971,-0.004916,-0.004707,-0.004266,-0.003803,-0.003557,-0.003082,-0.002877,-0.002551,-0.002378,-0.002035,-0.001643,-0.001214,-0.000852,-0.00051,0,0.000376,0.000647,0.001114,0.001418,0.001785,0.001965,0.002243,0.002337,0.002423,0.00246
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/OPTAAD/CGINS-OPTAAD-00193__20200302__CC_taarray.ext:85]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00193__20200302__CC_tcarray.ext [hunk 16]
```diff
@@ -0,0 +1,85 @@
...
+0.0045,0.004405,0.004246,0.003923,0.003679,0.003562,0.003402,0.003171,0.002796,0.002552,0.002587,0.002248,0.00211,0.002008,0.001883,0.001614,0.001481,0.001298,0.00105,0.000871,0.000811,0.000632,0.00044,0.000131,0,-0.000315,-0.000504,-0.00088,-0.001088,-0.0013,-0.001551,-0.001788,-0.002154,-0.002384,-0.002151
+0.004414,0.004564,0.004435,0.003976,0.003932,0.003874,0.003629,0.003568,0.003139,0.00301,0.002972,0.002555,0.002506,0.002224,0.002317,0.001914,0.001595,0.001476,0.001119,0.000877,0.000977,0.000755,0.000372,0.000157,0,-0.000157,-0.000411,-0.000771,-0.001082,-0.001351,-0.001696,-0.001947,-0.002371,-0.002737,-0.003506
+0.004783,0.004353,0.003968,0.003844,0.003801,0.003513,0.00308,0.00296,0.002824,0.002366,0.002466,0.00216,0.001938,0.001729,0.0018,0.001283,0.001352,0.001081,0.000864,0.000628,0.000671,0.000303,0.000125,5.1e-05,0,-0.000445,-0.000511,-0.000841,-0.001136,-0.001388,-0.001638,-0.001889,-0.002199,-0.002535,-0.002798
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/OPTAAD/CGINS-OPTAAD-00193__20200302__CC_tcarray.ext:85]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00152__20200328__CC_taarray.ext [hunk 17]
```diff
@@ -0,0 +1,83 @@
...
+-0.018754,-0.011507,-0.008569,-0.007012,-0.006178,-0.005291,-0.004479,-0.003395,-0.002321,-0.001152,-0.00019,0.000151,0.000413,0.000257,0.000359,0.000315,0.000175,0.000141,-3.5e-05,2e-05,-6.8e-05,2e-05,-0.000123,-6.7e-05,0,8e-05,8.7e-05,0.000137,-9.3e-05,-0.000141,-0.000426,-0.00052,-0.000844,-0.001163
+-0.018705,-0.01142,-0.008531,-0.006995,-0.006245,-0.005277,-0.004426,-0.003281,-0.002159,-0.000977,-7e-05,0.00022,0.000513,0.000273,0.000368,0.000329,0.000262,0.000333,6e-06,0.000185,7.3e-05,7.8e-05,8.2e-05,7e-06,0,0.000125,0.000193,0.000245,3.6e-05,4e-06,-0.000294,-0.000408,-0.000795,-0.001048
+-0.018826,-0.011458,-0.008642,-0.007111,-0.006234,-0.005252,-0.004498,-0.003356,-0.00219,-0.001074,-0.000135,0.000198,0.000509,0.00022,0.000439,0.000355,0.000293,0.000242,5.1e-05,0.0001,5.5e-05,8.8e-05,8.7e-05,6.3e-05,0,0.000254,0.000169,0.00028,-3.1e-05,-9.3e-05,-0.000254,-0.000436,-0.000831,-0.001166
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/OPTAAD/CGINS-OPTAAD-00152__20200328__CC_taarray.ext:83]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00152__20200328__CC_tcarray.ext [hunk 18]
```diff
@@ -0,0 +1,83 @@
...
+-0.004891,-0.000696,-4.6e-05,7.8e-05,0.000181,0.000148,0.000436,0.000657,0.000768,0.00118,0.001202,0.001271,0.001266,0.001215,0.001035,0.001102,0.001038,0.000886,0.000744,0.000643,0.000421,0.000284,9.6e-05,0.000185,0,-0.000315,-0.000503,-0.000798,-0.001219,-0.001579,-0.002009,-0.002384,-0.002777,-0.003383
+-0.005321,-0.001086,-0.000347,-0.000113,-0.00014,-0.000111,0.000144,0.000422,0.000559,0.000891,0.001092,0.001165,0.000981,0.00095,0.000841,0.000668,0.000749,0.000775,0.000753,0.00045,0.000428,0.000361,0.000141,1.7e-05,0,-0.000208,-0.000469,-0.000773,-0.001175,-0.001523,-0.001881,-0.002268,-0.002717,-0.003232
+-0.005686,-0.001182,-0.000397,-0.000301,-0.000403,-0.000259,0.000237,0.000377,0.000357,0.000814,0.000876,0.00112,0.001148,0.00133,0.001508,0.001226,0.000968,0.000829,0.000704,0.000528,0.000529,0.00029,-1.8e-05,0.000132,0,3.4e-05,-0.000307,-0.000584,-0.001142,-0.001257,-0.001846,-0.002098,-0.002475,-0.00273
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/OPTAAD/CGINS-OPTAAD-00152__20200328__CC_tcarray.ext:83]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00129__20200122__CC_taarray.ext [hunk 19]
```diff
@@ -0,0 +1,84 @@
...
+-0.008495,-0.006017,-0.004574,-0.00394,-0.003593,-0.003443,-0.003141,-0.002881,-0.002555,-0.002199,-0.00203,-0.001622,-0.0014,-0.001255,-0.001125,-0.000871,-0.000736,-0.000682,-0.00051,-0.00036,-0.000309,-5.7e-05,-2.8e-05,0,0.000141,0.000196,0.000308,0.000578,0.000492,0.000621,0.000809,0.000835,0.00098
+-0.008595,-0.006106,-0.004665,-0.004047,-0.003647,-0.003574,-0.00319,-0.00288,-0.002558,-0.002263,-0.002101,-0.001593,-0.00141,-0.001241,-0.001135,-0.000883,-0.000721,-0.000691,-0.000488,-0.000386,-0.000345,-0.000141,9e-06,0,0.000206,0.00026,0.000371,0.000645,0.000592,0.000663,0.000951,0.000944,0.001067
+-0.008575,-0.006178,-0.004739,-0.004118,-0.003685,-0.003632,-0.003346,-0.002959,-0.002668,-0.002334,-0.002171,-0.001724,-0.001598,-0.001324,-0.001233,-0.000948,-0.000879,-0.000805,-0.000512,-0.000352,-0.000398,-8.8e-05,-5.4e-05,0,0.000117,0.000269,0.000377,0.000626,0.000626,0.000713,0.00095,0.001011,0.001156
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/OPTAAD/CGINS-OPTAAD-00129__20200122__CC_taarray.ext:84]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00129__20200122__CC_tcarray.ext [hunk 20]
```diff
@@ -0,0 +1,84 @@
...
+-0.011231,-0.008599,-0.007425,-0.006846,-0.006336,-0.005951,-0.005565,-0.005056,-0.004638,-0.004237,-0.00392,-0.003441,-0.003125,-0.002759,-0.002315,-0.002134,-0.001773,-0.0016,-0.001226,-0.001009,-0.00095,-0.000524,-0.000254,0,7.5e-05,0.000435,0.0008,0.000873,0.000941,0.001289,0.001475,0.001596,0.001613
+-0.011567,-0.008869,-0.007751,-0.00697,-0.006479,-0.006024,-0.005414,-0.005023,-0.004609,-0.004148,-0.003754,-0.003302,-0.003105,-0.002554,-0.002206,-0.001992,-0.001692,-0.001514,-0.001093,-0.000915,-0.000742,-0.000644,-0.000119,0,0.000101,0.000352,0.00079,0.000757,0.00107,0.001298,0.001505,0.001672,0.001776
+-0.01185,-0.009018,-0.007657,-0.007084,-0.006392,-0.005883,-0.005508,-0.004981,-0.004717,-0.004245,-0.004055,-0.003367,-0.003214,-0.00289,-0.002402,-0.002208,-0.001874,-0.001577,-0.001415,-0.001059,-0.000847,-0.000493,-0.000217,0,0.000153,0.000611,0.0008,0.000971,0.001022,0.001349,0.001743,0.001818,0.001873
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/OPTAAD/CGINS-OPTAAD-00129__20200122__CC_tcarray.ext:84]: Looks good

**Pull request merged**